php/Terminus/Dispatcher/Subcommand.php

```diff
@@ -166,10 +167,10 @@ private function validate_args( $args, $assoc_args, $extra_args ) {
 
     $cmd_path = implode( ' ', get_path( $this ) );
     foreach ( $validator->get_unknown() as $token ) {
-      \Terminus::warning( sprintf(
-        "The `%s` command has an invalid synopsis part: %s",
-        $cmd_path, $token
-      ) );
+      \Terminus::get_logger()->warning(
```

Comment:
I have added a PR to fix the php 5.3 compatibility issues: #541 

That fix probably shouldn't wait for this beast to get cleaned up and merged.